data/labels.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "title": "vrger",
+    "color": "vsrgew"
+  },
```

Comment:
- [ ] According to the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png), the label should have an id. This is to ensure that each label is uniquely created ☺️.